[2025-05-31T12:34:40.743+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-31T12:34:40.758+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_daily_pollution_weather_dag.fetch_daily_data_all_cities scheduled__2025-05-29T00:00:00+00:00 [queued]>
[2025-05-31T12:34:40.761+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_daily_pollution_weather_dag.fetch_daily_data_all_cities scheduled__2025-05-29T00:00:00+00:00 [queued]>
[2025-05-31T12:34:40.761+0200] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2025-05-31T12:34:40.777+0200] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_daily_data_all_cities> on 2025-05-29 00:00:00+00:00
[2025-05-31T12:34:40.779+0200] {standard_task_runner.py:63} INFO - Started process 17127 to run task
[2025-05-31T12:34:40.783+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_daily_pollution_weather_dag', 'fetch_daily_data_all_cities', 'scheduled__2025-05-29T00:00:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/daily_dag.py', '--cfg-path', '/tmp/tmpj8mkqxyk']
[2025-05-31T12:34:40.783+0200] {standard_task_runner.py:91} INFO - Job 148: Subtask fetch_daily_data_all_cities
[2025-05-31T12:34:40.815+0200] {task_command.py:426} INFO - Running <TaskInstance: fetch_daily_pollution_weather_dag.fetch_daily_data_all_cities scheduled__2025-05-29T00:00:00+00:00 [running]> on host DESKTOP-2QS8J6D.
[2025-05-31T12:34:40.856+0200] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_daily_pollution_weather_dag' AIRFLOW_CTX_TASK_ID='fetch_daily_data_all_cities' AIRFLOW_CTX_EXECUTION_DATE='2025-05-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-29T00:00:00+00:00'
[2025-05-31T12:34:40.856+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-31T12:37:24.740+0200] {python.py:237} INFO - Done. Returned value was: None
[2025-05-31T12:37:24.740+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-31T12:37:24.743+0200] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_daily_pollution_weather_dag, task_id=fetch_daily_data_all_cities, execution_date=20250529T000000, start_date=20250531T103440, end_date=20250531T103724
[2025-05-31T12:37:24.778+0200] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-31T12:37:24.786+0200] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-31T12:37:24.805+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-05-31T13:24:55.583+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-31T13:24:55.602+0200] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: fetch_daily_pollution_weather_dag.fetch_daily_data_all_cities scheduled__2025-05-29T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2025-05-31T11:24:55.602408+00:00 and task will be retried at 2025-05-31T11:29:51.763134+00:00.
[2025-05-31T13:24:55.602+0200] {local_task_job_runner.py:163} INFO - Task is not able to be run
[2025-05-31T14:06:17.781+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-31T14:06:17.792+0200] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: fetch_daily_pollution_weather_dag.fetch_daily_data_all_cities scheduled__2025-05-29T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2025-05-31T12:06:17.792815+00:00 and task will be retried at 2025-05-31T12:11:13.326523+00:00.
[2025-05-31T14:06:17.795+0200] {local_task_job_runner.py:163} INFO - Task is not able to be run
